src/components/combo_box/combo_box.tsx

```diff
@@ -126,6 +126,7 @@ interface _EuiComboBoxProps<T>
    * `string` | `ReactElement` or an array of these
    */
   append?: EuiFormControlLayoutProps['append'];
+  delimiter?: string;
```

Comment:
Needs a prop comment
```suggestion
  /**
   * A special character to use as a value separator. Typically a comma `,`
   */
  delimiter?: string;
```